front/src/components/BoxReconciliationOverlay/components/MatchProductsForm.tsx

```diff
@@ -155,14 +155,18 @@ export function MatchProductsForm({
     <form onSubmit={handleSubmit(onSubmitMatchProductsForm)}>
       <Flex direction="column" gap="2" alignItems="stretch">
```

Comment:
Nice to have: increase the gap to 4 and put a Box around the Product Gender part and one around the Size. Looks a bit nicer